DER/DerivativesContracts/Options.rdf

```diff
@@ -254,16 +321,139 @@
 		<owl:disjointWith rdf:resource="&fibo-der-der-opt;LinearPayoutCommitment"/>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-der-opt;CurrencyOptionExerciseTerms">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ex;ExerciseTerms"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-der-opt;delivery"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;AmountOfMoney"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">currency option exercise terms</rdfs:label>
+		<skos:definition xml:lang="en">Contractual terms defining when or under what circumstances an currency option is to be exercised.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Also includes terms defining delivery when the contract is exercised. Delivery for currency options is by way of cash.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
 	<owl:Class rdf:about="&fibo-der-der-opt;DefinedConditionsPayoutCommitment">
 		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;OptionPayoutCommitment"/>
 		<rdfs:label xml:lang="en">defined conditions payout commitment</rdfs:label>
 		<skos:definition xml:lang="en">A commitment which is only made when certain defined conditions are met. Further notes: Like all contractual commitments, the details of the commitment (and in this case, the conditions) are defined in the corresponding Contractual Terms. These set out what the commitment is and what the conditions are under which it is to be met. Term introduced in order to support digital and other exotic options. Note this is similar to but distinct from commitments where the specific nature of the commitment (typically a cashflow) is defined by resolving some mathematical formula i.e. where the commitment to do something is not conditional but the precise monetary amounts (payment, settlement) are.</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;EarlyExerciseCommitment">
-		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;OptionWriterCommitment"/>
-		<rdfs:label xml:lang="en">early exercise commitment</rdfs:label>
-		<skos:definition xml:lang="en">The commitment to exercise the option ahead of the expiry date under given conditions as laid out in the Barrier Feature terms of the Option contract.</skos:definition>
+	<owl:Class rdf:about="&fibo-der-der-opt;EquityIndexOptionUnderlying">
```

Comment:
don't know why this class is needed. Or even Underlier as a class - why not hasUnderlier property from the option directly to the instrument?